maas-region/src/helper.py

```diff
@@ -194,6 +194,22 @@ def setup_region(maas_url: str, dsn: str, mode: str) -> None:
         ]
         subprocess.check_call(cmd)
 
+    @staticmethod
+    def is_tls_enabled() -> bool | None:
+        """Check whether MAAS currently has TLS enabled.
+
+        Returns:
+            bool | None: True if MAAS has TLS enabled, False if not, None if MAAS is not initialized
+        """
+        maas = SnapCache()[MAAS_SNAP_NAME]
+        nginx_cfg_filename = f"/var/snap/maas/{maas.revision}/http/regiond.nginx.conf"
+        try:
+            with open(nginx_cfg_filename, "r") as f:
+                return "listen 5443" in f.read()
```

Comment:
```suggestion
                return f"listen {MAAS_HTTPS_PORT}" in f.read()
```

Reply:
This would require a circular import chain between helper.py and charm.py, unless we duplicate the `MAAS_HTTPS_PORT` variable to helper.py (Which I will do for now, since I think that still makes more sense than hard-coding the number here)